### PR TITLE
docs(modal): document onInit

### PR DESCRIPTION
**Pull request by Foxandxss**

### CODE REVIEW

File: src/modal/docs/readme.md [hunk 1]
```diff
@@ -119,6 +119,8 @@ The scope associated with modal's content is augmented with:
 
 Those methods make it easy to close a modal window without a need to create a dedicated controller.
 
+Also, when using `bindToController` you can define a `$onInit` method in the controller that will fire upon initialization.
```
Comment (icfantv) [inline src/modal/docs/readme.md]: - Comma after `bindToController`
- An `$onInit` method.

**Comment by icfantv** [comment]
Two tweaks and then LGTM.

File: src/modal/docs/readme.md (hunk 1 above)
Comment (Foxandxss) [inline src/modal/docs/readme.md]: yeah, those are the kind that I don't forget, hah.

**Comment by icfantv** [comment]
LGTM.

**Pull request closed without merging**